telemetry/definitions/commonDefinitions.json

```diff
@@ -257,6 +257,17 @@
                 "local"
             ]
         },
+        {
+            "name": "codeFixAction",
+            "type": "string",
+            "description": "Captures the type of fix that was accepted",
+            "allowedValues": [
+                "openDiff",
+                "insertAtCursor",
+                "copyDiff",
+                "quickFix"
```

Comment:
If I understand correctly, we will use `codewhisperer_codeScanIssueApplyFix` for both applying the fix and actions _before_ applying the fix, is that right?

Reply:
yes